app/assets/javascripts/index/element.js

```diff
@@ -101,4 +110,97 @@
       scrollableList.scrollLeft = scrollableList.scrollWidth - scrollableList.offsetWidth;
     }
   }
+
+  function previewWikidataValue($btn) {
+    if (!OSM.WIKIDATA_API_URL) return;
+    const items = $btn.data("qids");
+    if (!items?.length) return;
+    $btn.addClass("disabled").removeClass("wdt-preview");
+    fetch(OSM.WIKIDATA_API_URL + "?" + new URLSearchParams({
+      action: "wbgetentities",
+      format: "json",
+      origin: "*",
+      ids: items.join("|"),
+      props: "labels|sitelinks|claims|descriptions",
```

Comment:
We can request `sitelinks/urls` here, so that each object in `sitelinks` will contain a `url` property set to the absolute URL to the article.